src/pseudopeople/configuration/interface.py

```diff
@@ -14,26 +14,25 @@ def get_config(dataset_name: str = None, overrides: Union[Path, str, Dict] = Non
     """
     Function that returns the pseudopeople configuration,
     including all default values.
-    If :code:`dataset_name` is None (the default), the returned dictionary has exactly
+    If :code:`dataset_name` is None (the default), the returned dictionary includes
+    the configuration for all datasets. If a dataset name is supplied, only that
+    dataset's configuration is returned. In both cases, the returned dictionary has exactly
     the structure described on the :ref:`Configuration page <configuration_main>`.
-    If a dataset name is supplied, only returns the part of the configuration for that
-    dataset, i.e. :code:`get_config(dataset_name)` is equivalent to
-    :code:`get_config()[dataset_name]`.
 
     To get the default probability of nonresponse in the Decennial Census dataset:
 
     .. code-block:: pycon
 
         >>> import pseudopeople as psp
-        >>> psp.get_config('decennial_census')['row_noise']['do_not_respond']
+        >>> psp.get_config('decennial_census')['decennial_census']['row_noise']['do_not_respond']
         {'row_probability': 0.0145}
 
     To view that same part of the configuration after applying a user override:
 
     .. code-block:: pycon
 
         >>> overrides = {'decennial_census': {'row_noise': {'do_not_respond': {'row_probability': 0.1}}}}
-        >>> psp.get_config('decennial_census', overrides)['row_noise']['do_not_respond']
+        >>> psp.get_config('decennial_census', overrides)['decennial_census']['row_noise']['do_not_respond']
```

Comment:
Same comment here -- this could also be achieved with `psp.get_config(overrides=overrides)['decennial_census']['row_noise']['do_not_respond']`

Reply:
My previous thinking is that it would be convenient to have just the stripped down dict for a single dataset.  but even the stripped down dict is overwhelming, so now I'm leaning towards simplifying the function and not having the dataset argument.

Reply:
I think having this parameter was kind of pointless before we updated `get_config`, because it's just as easy to do `get_config()[dataset_name]` as `get_config(dataset_name)`. But now it actually saves you some code if you want to work with a configuration for a single dataset, since you don't manually have to add the outer-level key.